src/utils/SdkCoin.sol

```diff
@@ -48,7 +48,7 @@ library SdkCoin {
      * @return convertedAmount The amount converted to uint64 supported by cosmos coins
      * @return remainder The remainder of the conversion
      */
-    function _ERC20ToSdkCoin_ConvertAmount(
+    function convertAmountERC20ToSdkCoin(
```

Comment:
Do you think we should name these `convertAmountFromERC20` and `convertAmountToERC20` so that it is not so long?

```suggestion
    function convertAmountFromERC20(
```

Reply:
Ok, I prefer the shorter one because SdkCoin is already the name of the library. So it will look like `SdkCoin.convertAmountFromERC20`

Reply:
Makes sense! Please @MukulKolpe, do you mind taking into account @srdtrk suggestion and apply the changes needed in all the files?

Reply:
Agree on the `SdkCoin.convertAmountFromERC20` name

Reply:
Thank you! I’ve applied the suggested changes across all the files. Please let me know if there are any further suggestions.